mysql-test/suite/innodb/t/ts_encrypt.test

```diff
@@ -0,0 +1,174 @@
+# InnoDB general tablespace encryption
+
+call mtr.add_suppression("Partition.*does not exist in the InnoDB internal data dictionary though");
```

Comment:
Please, add more comments to the test.
Just adding section names would make it much more readable. Say, "changing tablespace encryption oprions", "moving table to another tablespace", "partitions", etc.